Source/MayaLiveLinkPlugin.cpp

```diff
@@ -466,6 +672,7 @@ struct FLiveLinkStreamedJointHierarchySubject : IStreamedEntity
 	FName SubjectName;
 	MDagPath RootDagPath;
 	TArray<FStreamHierarchy> JointsToStream;
+	FStreamedUserDefinedAttributes StreamedAttributes;
```

Comment:
Having support for custom attributes across all type of streamed object will be preferable.

Reply:
We won't have time internally to fix this feature across all types of streamed subjects (props, camera, lights, skeleton, joints). To accept that PR "as is" it would need to be supported on any objects. Listening for objects changed / attribute changed instead of constantly polling for changes would ease the process of detecting when attributes are dirty. Feel free to update the pull request if desired.

Reply:
My experience with the Maya API is very sparse and I don't believe I will be able to make a second pass on the code in quite a while. (due to workload) It is likely that you will release the update before I look at this again.

Reply:
No worries, I just wanted to outline that we would not have time internally to wrap it up correctly. Thanks for your input!